docs/docs/usage/dependency.md

```diff
@@ -77,6 +97,29 @@ To update the specified package(s):
 $ pdm update requests
 ```
 
+To update multiple sections of dependencies:
+
+```console
+$ pdm update -s security -s http
+```
+
+To update a given package in the specified section:
+
+```console
+$ pdm update -s security cryptography
+```
+
+If the section is not given, PDM will search for the requirement in the default dependencies set and raises an error if none is found.
+
+To update package in development dependencies:
+
+```console
+# Update all dev-dependencies
+$ pdm update -d
+# Update the specified group of dev-dependencies
+$ pdm update -ds test pytest
```

Comment:
I think you meant `pdm update -ds test` (without `pytest`)?

Reply:
No, you can give package name(s) to update command to tell PDM only to update those given packages under the given group(Yes, group should be given or it defaults to the default group, as the same package can appear in more than 1 group)

Reply:
May I suggest something like this then:

```
# Update a package in the specified group of dev-dependencies
```